src/store/getters/state-getters.ts

```diff
@@ -644,3 +650,11 @@ export const isDissolutionDefineDissolutionValid = (state: StateIF): boolean =>
 export const getRegistration = (state: StateIF): RegistrationStateIF => {
   return state.stateModel.registration
 }
+
+// current filing subtitle
+export const getFilingSubtitle = (state: StateIF): string => {
+  if (getEntityType(state) === CorpTypeCd.SOLE_PROP) {
+    return FilingTypesSubTitle.SOLE_PROP_SUB_TITLE
+  }
```

Comment:
This does not apply to all, say, SP filings. If we merge this code then SP registration filings will incorrectly display this subtitle.

So the filing type needs to be taken into account.